### DISCUSSION TITLE
Create multi volume error

### DISCUSSION THREAD

**Discussion by ethuwlwfu3288** [juicedata/juicefs#3927] [General]
**What happened**:

I use format to create volume dev1, then I want to use format to create volume dev2, but get error 

juicefs format     --storage cos     --bucket https://bucket1.cos.ap-guangzhou.myqcloud.com     --access-key xxx     --secret-key xxx  --no-update true    "mysql://xx:xx@(ip:3306)/juicefs"    dev1

juicefs format     --storage cos     --bucket https://bucket1.cos.ap-guangzhou.myqcloud.com     --access-key xxx     --secret-key xxx  --no-update true    "mysql://xx:xx@(ip:3306)/juicefs"    dev2

juicefs[63272] <FATAL>: format: cannot update volume name from dev1 to dev2 [format.go:469]

Then i tried format --no-update option, it doesn't work.

**What you expected to happen**:

I expect to create multi volume to make a Multi-Tenant storage system.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

![image](https://github.com/juicedata/juicefs/assets/132146847/46b8c21a-682c-44c5-bd2f-5b4a826fa64b)

I find this case in doc, but I don't know how to clean up the metadata engine without lose meta data.

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version: juicefs version 1.0.4+2023-04-06.f1c475d
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by SandyXSD** [comment]
You can create at most ONE volume for each database. So pick another database/schema in MySQL as the metadata engine for dev2.
